Buildings/Experimental/DHC/EnergyTransferStations/Cooling/DirectControlled.mo

```diff
@@ -0,0 +1,300 @@
+    redeclare final package Medium=MediumSer,
+    final energyDynamics=energyDynamics,
+    final m_flow_nominal={mBui_flow_nominal,-mDis_flow_nominal,-mByp_flow_nominal},
+    final dp_nominal=dp_nominal)
+    "Bypass junction, splitter"
+    annotation (Placement(transformation(extent={{-60,190},{-40,210}})));
+  Buildings.Fluid.Actuators.Valves.TwoWayEqualPercentage conVal(
+    redeclare final package Medium=MediumSer,
+    final m_flow_nominal=mDis_flow_nominal,
+    final dpValve_nominal=dpConVal_nominal,
+    use_inputFilter=true,
+    riseTime(displayUnit="s") = 60)
+    "Control valve"
+    annotation (Placement(transformation(extent={{-10,210},{10,190}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTBuiRet(
+    redeclare final package Medium = MediumSer,
+    final m_flow_nominal=mBui_flow_nominal)
+    "Building return temperature sensor"
+    annotation (Placement(transformation(extent={{-220,210},{-200,190}})));
+  Modelica.Fluid.Valves.ValveIncompressible cheVal(
+    redeclare final package Medium=MediumSer,
+    final allowFlowReversal=false,
+    final dp_nominal=dpCheVal_nominal,
+    final m_flow_nominal=mByp_flow_nominal,
+    final filteredOpening=true,
+    riseTime(
+      displayUnit="s")=60,
+    final checkValve=true)
+    "Check valve (backflow preventer)"
+    annotation (Placement(transformation(extent={{10,-10},{-10,10}},
+        rotation=90,
+        origin={-50,-10})));
+  Modelica.Blocks.Math.Add dTdis(
+    final k1=-1)
+    "Temperature difference on the district side"
+    annotation (Placement(transformation(extent={{60,-114},{80,-94}})));
+  Modelica.Blocks.Math.Product pro
+    "Product"
+    annotation (Placement(transformation(extent={{120,-120},{140,-100}})));
+  Modelica.Blocks.Math.Gain cp(
+    final k=cp_default)
+    "Specific heat multiplier to calculate heat flow rate"
+    annotation (Placement(transformation(extent={{180,-120},{200,-100}})));
+  Modelica.Blocks.Continuous.Integrator int(
+    final k=1)
+    "Integration"
+    annotation (Placement(transformation(extent={{260,-170},{280,-150}})));
+  Modelica.Blocks.Sources.Constant ope(
+    final k=1)
+    "Check valve is always open in the positive flow direction"
+    annotation (Placement(transformation(extent={{-100,-20},{-80,0}})));
+  Buildings.Controls.Continuous.LimPID con(
+    final controllerType=controllerType,
+    final k=k,
+    final Ti=Ti,
+    final Td=Td,
+    final yMax=yMax,
+    final yMin=yMin,
+    reverseActing=false,
+    y_reset=0)
+    "District return temperature controller"
+    annotation (Placement(transformation(extent={{-220,10},{-200,-10}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTBuiSup(
+    redeclare final package Medium = MediumSer,
+    final m_flow_nominal=mBui_flow_nominal)
+    "Building supply temperature sensor"
+    annotation (Placement(transformation(extent={{230,190},{250,210}})));
+  Modelica.Blocks.Logical.OnOffController onOffCon(bandwidth=0.2)
```

Comment:
propagate the `bandwidth` to be an advanced parameter. Finalized the parameter assignment.

Reply:
Done